scripts/run-sauce-tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
```

Comment:
Why do you need to flip the chmod flags?

Reply:
These aren't `chmod` flags. These set flags tell the shell to:

 1. Exit on the first non-zero exit code (otherwise it would invoke all commands
 2. Print all the commands being run by the script as they are invoked

See https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#The-Set-Builtin